### PR TITLE
pull request 

### PR DESCRIPTION
**Pull request by Oberiin**

### CODE REVIEW

**Comment by A-Ostrovnyy** [comment]
One of the main rules of this PR is don't touch the changes of other students, Please restore changes from other students and add just yours :grinning: 
[Use this command](https://git-scm.com/docs/git-revert) to restore changes

**Comment by Oberiin** [comment]
Lol, sorry, I'll restore changes.

Thanks!

**Comment by suchov** [comment]
Tricky task, huh?

File: README.md [hunk 1]
```diff
@@ -102,7 +106,6 @@ I am a freelancer, designer and developer. I love learning, and I hope to get ne
 
 My name is Aleksandr, very glad to join you!
 
-
```
Comment (suchov) [inline README.md:105]: no need for this from your IDE I assume

File: README.md [hunk 2]
```diff
@@ -65,7 +69,7 @@ Great and importand job 👍
 
 Let`s continue. Good luck everyone
 
-Hello!
```
Comment (suchov) [inline README.md:68]: is it on purpose?

**Review by suchov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**